cloud-controller-manager/do/loadbalancers.go

```diff
@@ -1275,6 +1286,16 @@ func getDisownLB(service *v1.Service) (bool, error) {
 	return disownLB, nil
 }
 
+// getAttachedLB returns whether the load-balancer was attached to the service..
+// False is returned if not specified.
+func getAttachedLB(service *v1.Service) (bool, error) {
+	attachedLB, _, err := getBool(service.Annotations, annoDOLoadBalancerID)
```

Comment:
`annoDOLoadBalancerID` is managed by our CCM to more easily and directly look up LBs in the DO API. I'm not sure I understand why we should stop deleting LBs for which CCM added the annotation in the first place?

Reply:
I also feel we should have a new annotation to preserve LBs.

If it's about separating LBs managed by different CCM implementations, then [the LB class](https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class) introduced in 1.24 might be worth considering adding support for in our CCM.

Reply:
It's not different CCM implementations. My Pulumi code provisions a DO LB and adds domain names to it for TLS, and then in my LB service I provide the load-balancer-id annotation to hook them up.

Reply:
I see. A separate, new annotation would seem appropriate to me then.

Reply:
Pushed as new annotation